docs/authenticated-pages.md

```diff
@@ -0,0 +1,33 @@
+# Running Lighthouse on Authenticated Pages
+
+Default runs of Lighthouse load a page as a "new user", with no previous session or storage data. This means that pages requiring authenticated access do not work without additional setup.
+
+## Puppeteer
+
+[Puppeteer](https://pptr.dev) is the most flexible approach for auditing pages behind authentication with Lighthouse. See [recipes/auth](./recipes/auth) for more.
+
+## Chrome DevTools
+
+The Audits panel in Chrome DevTools will never clear your session cookies, so you can log in to the target site and run Lighthouse without being logged out. If `localStorage` or `indexedDB` is important for your authentication purposes, be sure to uncheck `Clear storage`.
+
+## Headers
+
+CLI:
+```sh
+lighthouse http://www.example.com --view --extra-headers="{\"Authorization\":\"...\"}"
+```
+
+Node:
+```js
+const result = await lighthouse('http://www.example.com', {
+  extraHeaders: {
+    Authorization: '...',
+  },
+});
+```
+
+You could also set the `Cookie` header, but beware: it will [override any other Cookies you expect to be there](https://github.com/GoogleChrome/lighthouse/pull/9170). A workaround is to use Puppeteer's [`page.setCookie`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetcookiecookies).
+
+## Chrome User Profile
+
+TODO: pending [#8957](https://github.com/GoogleChrome/lighthouse/issues/8957).
```

Comment:
TODOs in the docs make the doc writers look bad when really in this case it's a functionality TODO and when it's done the docs can follow :)

Maybe after mentioning the existing workflow above, it could say that an easier method is being considered for implementation and you can learn more at [issue link] (and that the reader can weigh in there)? I'm not sure how much detail is important to include here vs redirecting over there